demo/chatinterface_multimodal/run.py

```diff
@@ -5,7 +5,7 @@ def echo(message, history):
 
 demo = gr.ChatInterface(
     fn=echo,
-    examples=[{"text": "hello"}, {"text": "hola"}, {"text": "merhaba"}],
+    examples=[{"icon": "files/avatar.png", "display_text": "Display Text Here!", "text": "Try this example with this audio.", "files": ["files/cantina.wav"]}, {"text": "Try this example with this image.", "files": ["files/avatar.png"]}, {"text": "This is just text, no files!"}, {"text": "Try this example with this image.", "files": ["files/avatar.png", "files/avatar.png"]}, {"text": "Try this example with this Audio.", "files": ["files/cantina.wav"]}],
```

Comment:
nit but I would use the full path here otherwise you have to run the demo from this directory. or make a note that it must be run for this directory